### ISSUE TITLE
Automate maintainers comment on new issues

### ISSUE THREAD

**Issue by AlbertoPdRF** [forem/forem#9442]
**Is your feature request related to a problem? Please describe.**

When a new issue is opened, a maintainer manually comments it with a template. That's some annoying manual work that could be easily avoided!

**Describe the solution you'd like**

A GitHub Action that automatically posts the comment.

**Describe alternatives you've considered**

Keep doing the work manually.

**Additional context**

I've briefly discussed this with @rhymes and one downside is that the template comment doesn't get posted on every issue, but we think that manually superseding the automatic comment on some issues is more convenient than having to manually post the comment on all of them.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @forem/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/forem/forem/blob/master/CONTRIBUTING.md).

**Comment by Amorpheuz** [comment]
I would love to work on this if possible! I have been recently experimenting a ton with GitHub actions and workflows.

**Comment by rhymes** [comment]
@Amorpheuz hi! I'm sorry if I didn't assign this sooner but @AlbertoPdRF expressed the intention to write the implementation of it. Maybe, if it's doable you can use this or other means of communication to coordinate a solution?

The purpose here is to leave a comment like https://github.com/forem/forem/issues/9442#issuecomment-662488130 under the initial post in a issue.

The markdown is the following:

```markdown
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @forem/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/forem/forem/blob/master/CONTRIBUTING.md).
```

**Comment by Amorpheuz** [comment]
Ahh! No problem, I had my suspicions that it was more of a picked-up issue, haha. But gave it a shot since it wasn't assigned!

Awesome idea @AlbertoPdRF, feel free to hit me up if you hit some help.

**Comment by AlbertoPdRF** [comment]
> Awesome idea @AlbertoPdRF, feel free to hit me up if you need some help.

Will do, thanks for the kind offer! 🙂
